runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.service.ts

```diff
@@ -72,9 +73,24 @@ export class CredentialService {
   }
 
   async findAllAccessible(user: IUser) {
-    const accessibleCollections = await this.collectionService.findAllAccessible(user);
+    const accessibleCollections = await this.collectionService.findAllAccessibleWithUser(user);
 
-    const credentials = accessibleCollections.map((collection) => collection.credentials).flat();
+    if (accessibleCollections.length === 0) {
+      throw new NotFoundException('Could not find any accessible collections');
+    }
```

Comment:
should we really throw an error or return empty array and on the front if array is empty display not error but UI message "No collections found"?

Reply:
I could return here empty array